src/app/forms/app-for-coe-with-release/app-for-coe-with-release.web.component.js

```diff
@@ -25,126 +25,67 @@ export default class AppForCOENoFees_Web extends React.Component {
             dataKey="person.coeRecipient"
             label={__("coe recipient explanation")}
             options={petitionerRepresentativeOptions}
-            {...this.props}
           />
           {this.props.data.person.coeRecipient ===
             "someone-else-is-recipient" && (
             <>
               <TextInput
                 dataKey="person.coeRecipientName"
                 label={__("coe recipient name")}
-                {...this.props}
               />
               <TextInput
                 dataKey="person.coeRecipientAgency"
                 label={__("coe recipient agency")}
-                {...this.props}
               />
               <TextInput
                 dataKey="person.coeRecipientStreet"
                 label={__("street")}
-                {...this.props}
-              />
-              <TextInput
-                dataKey="person.coeRecipientCity"
-                label={__("city")}
-                {...this.props}
               />
+              <TextInput dataKey="person.coeRecipientCity" label={__("city")} />
```

Comment:
Prettier changed this to one line instead of five lines and github's diff tool got confused.